recipes/encyclopedia/encyclopedia.py

```diff
@@ -14,8 +14,8 @@
 from os import getenv
 from os import X_OK
 
-# https://bitbucket.org/searleb/encyclopedia/downloads/encyclopedia-0.9.5-executable.jar
-jar_file = 'encyclopedia-0.9.5-executable.jar'
+# https://bitbucket.org/searleb/encyclopedia/downloads/encyclopedia-1.2.2-executable.jar
+jar_file = f'encyclopedia-{ getenv("PKG_VERSION") }-executable.jar'
```

Comment:
I have found the working workaround!
```suggestion
jar_file = 'encyclopedia-{}-executable.jar'.format(getenv('PKG_VERSION'))
```

Reply:
Maybe...